app/src/main/java/com/example/xpvehicles/activities/VehicleImageZoomInActivity.java

```diff
@@ -0,0 +1,46 @@
+package com.example.xpvehicles.activities;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.graphics.BitmapFactory;
+import android.os.Bundle;
+import android.view.MotionEvent;
+import android.view.ScaleGestureDetector;
+import android.widget.ImageView;
+
+import com.bumptech.glide.Glide;
+import com.example.xpvehicles.R;
+import com.google.android.material.appbar.MaterialToolbar;
+import com.ortiz.touchview.TouchImageView;
+import com.parse.ParseFile;
+
+public class VehicleImageZoomInActivity extends AppCompatActivity {
+
+    private static final String TAG = "VehicleImageZoomInActivity";
+    private MaterialToolbar topAppBarVehicleImageZoomIn;
+    private ParseFile image;
+    private TouchImageView touchImageViewVehicleImageZoom;
+
+    private void setTopAppBarOnClickListener() {
+        topAppBarVehicleImageZoomIn.setNavigationOnClickListener(v -> {
+            this.finish();
+        });
+    }
```

Comment:
I think I see this a couple times. You can probably create a base activity and use inheritance to avoid duplicate code?